rpc/codec/rpc_test.go

```diff
@@ -250,3 +253,93 @@ func testEchoClientAsync(t *testing.T, client *rpc.Client) {
 		)
 	}
 }
+
+func randString(n int) string {
```

Comment:
Yes, but nothing in this package depends on `cockroach/util`. Thought it was better to minimize dependencies.